README.md

```diff
@@ -17,38 +17,38 @@ So, let's do it!
+[VBTC Vietnam](https://vbtc.exchange/) | **YES** (JAN-2021) | :zap: | [Twitter](https://twitter.com/VBTC_Vietnam/status/1353564136702005248) | n.a. | n.a. | :broken_heart: YES | n.a.
+[FixedFloat](https://fixedfloat.com/) | **YES** (FEB-2019) | :zap: | [blog](https://fixedfloat.com/blog/currency/lightning-network) | n.a. | [Link](https://1ml.com/node/037f990e61acee8a7697966afd29dd88f3b1f8a7b14d625c4f8742bd952003a590) fixedfloat.com | :green_heart: NO | n.a.
+[LOFT](https://loft.trade/) | **YES** (FEB-2021) | :zap: | [Twitter](https://twitter.com/LoftTrade/status/1370047636728844288) | n.a. | n.a. | :green_heart: NO | n.a.
+[PrimeBit](https://primebit.com/) | **YES** (SEP-2021) | :zap: | [Twitter](https://twitter.com/primebit_com/status/1438078966384975872) | n.a. | n.a. | :green_heart: NO | n.a.
+[SimpleFX](https://simplefx.com/) | **YES** (SEP-2021) | :zap: | [website](https://simplefx.com/bitcoin-lightning-network/) | n.a. | n.a. | :green_heart: NO | n.a.
+[BitcoinVN](https://bitcoinvn.io) | **YES** (NOV-2021) | :zap: | [website](https://bitcoinvn.io/?deposit=btcln) | n.a. | n.a. | :green_heart: NO | n.a.
+[Bipa](https://bipa.app) | **YES** (NOV-2020) | :zap: | [blog](https://medium.com/o-blog-da-bipa/bipa-lan%C3%A7a-suporte-%C3%A0-lightning-network-d7474409059e) | n.a. | [Link](https://1ml.com/node/02fb79c3a9121d85b126687bd111eaebf21aaaaa5cbf232e2b6c3bdf8803f40182) bipa | :broken_heart: YES | n.a.
+[Boltz](https://boltz.exchange) | **YES** (APR-2019) | :zap: | [website](https://www.notion.so/Frequently-Asked-Questions-585328ae43944e2eba351050790d5eec) | n.a. | [Link](https://1ml.com/node/026165850492521f4ac8abd9bd8088123446d126f648ca35e60f88177dc149ceb2) boltz.exchange | :green_heart: NO | n.a.
+[Kollider](https://pro.kollider.xyz) | **YES** (JAN-2021) | :zap: | [Twitter](https://twitter.com/kollider_trade) | n.a. | [Link](https://1ml.com/node/02fe80fb6a2dc0fb6e9bec49c76d048889c91355d4e900fcb026bf095665790325) kollider| :green_heart: NO | 0.005
+[LN Markets](https://lnmarkets.com) | **YES** (MAR-2020) | :zap: | [Twitter](https://twitter.com/LNMarkets/status/1237778507062751232) | n.a. | [Link](https://amboss.space/node/03271338633d2d37b285dae4df40b413d8c6c791fbee7797bc5dc70812196d7d5c) LN Markets| :green_heart: NO | 0.05
+[Tauros](https://tauros.io) | **YES** (MAY-2022) | :zap: | [article](https://www.nasdaq.com/articles/tauros-ibex-mercado-partner-for-mexicos-first-lightning-enabled-bitcoin-exchange) | n.a. | n.a | n.a | n.a
+[Coinfinity](https://coinfinity.co) | **YES** (JUN-2023) | :hourglass_flowing_sand: | [Twitter](https://twitter.com/coinfinity/status/1663870230454476801) | n.a. | [Link](https://amboss.space/node/02d4531a2f2e6e5a9033d37d548cff4834a3898e74c3abe1985b493c42ebbd707d) coinfinity.co | :broken_heart: YES | n.a
```

Comment:
This has not the lightning symbol so it should be reverted to not bold

Reply:
good point! will address

Reply:
@theDavidCoen done